Source/DafnyCore/Backends/SinglePassCodeGenerator/SinglePassCodeGenerator.Statement.cs

```diff
@@ -594,6 +594,10 @@ public abstract partial class SinglePassCodeGenerator {
         writer = thenWriter;
       } else if (pattern is IdPattern idPattern) {
         if (idPattern.BoundVar != null) {
+          if (idPattern.BoundVar.Tok.val.StartsWith("_")) {
```

Comment:
Any chance you could replace `_` with a reference to a constant that's also used somewhere else? Same for the next usage of `_`.